# Constantes de localização
uint8 NORTE = 0
uint8 SUL = 1 
uint8 LESTE = 2
uint8 OESTE = 3
# Equivalentes às acima
uint8 ORIENTATION_TREM = 0
uint8 ORIENTATION_INICIO = 1
uint8 ORIENTATION_AZUL = 2
uint8 ORIENTATION_VERDE = 3
# Constantes de localização
uint8 QUADRANTE_CENTRAL=0
uint8 QUADRANTE_ESQUERDO=1
uint8 QUADRANTE_DIREITO=2
uint8 DOCA_VERDE=3
uint8 DOCA_AZUL=4
uint8 DOCA_CENTRAL=5
uint8 TREM=6
# Mensagem com localização e orientação
uint8 location
uint8 orientation
#Quantidade de containers depositados
uint8 code
